src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -240,7 +240,7 @@ object Serializer {
 
   private def s(node: DefModule)(implicit b: StringBuilder, indent: Int): Unit = node match {
     case Module(info, name, ports, body) =>
-      b ++= "module "; b ++= name; b ++= " :"; s(info)
+      doIndent(0); b ++= "module "; b ++= name; b ++= " :"; s(info)
       ports.foreach { p => newLineAndIndent(1); s(p) }
       newLineNoIndent() // add a new line between port declaration and body
       newLineAndIndent(1); s(body)(b, indent + 1)
```

Comment:
Basically just moving the indention to the module emission itself rather than inside of the `Circuit` emission.